plugins/chat/spec/system/page_objects/chat/chat_channel.rb

```diff
@@ -64,7 +64,9 @@ def expand_message_actions(message)
       end
 
       def expand_message_actions_mobile(message, delay: 2)
-        message_by_id(message.id).click(delay: delay)
+        within(message_by_id_selector(message.id)) do
```

Comment:
`within` we have found ends up randomly flaky too, just doing this is equivalent and _seems_ to be more consistent for whatever reason:

```ruby
find(message_by_id_selector(message.id)).find(".chat-message-content").click(delay: delay)
```

Reply:
@martin-brennan Done :+1:

Reply:
Can confirm that `within` does not do what you think it does..